index.js

```diff
@@ -27,6 +27,16 @@ function assign(a, b) {
       a[key] = b[key];
     }
   }
+
+  if (typeof Object.getOwnPropertySymbols === 'function') {
```

Comment:
you could use https://github.com/jonschlinkert/assign-symbols if you want. I don't mind either way, just wanted to mention it.